hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -333,10 +348,40 @@ def _apply_updates_impl(self, delta: tu.ThreatUpdatesDelta) -> None:
                 state.pop(item.key, None)
             else:
                 state[item.key] = item
+                updated[item.key] = item
 
         self._store_state(state.values())
         self._cached_state = state
 
+        post_apply_fn(updated)
+
+    def post_apply(self, updated: t.Dict = {}):
```

Comment:
same typing thing. Can we describe what the Dict is of

Reply:
This is the level of type granularity that is defined for `ThreatUpdateS3PDQStore._cached_state` 
If we want to refactor this further to for better type support I'm not opposed but this functions shouldn't (and I'm not sure mypy will let us) be more type restrictive than the class using it imo